### PR TITLE
feat: add default engine data

### PR DESCRIPTION
**Pull request by farzer**
#118

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -171,6 +172,7 @@ function fastifyView (fastify, opts, next) {
       return
     }
 
+    data = Object.assign(defaultCtx, data)
```
Comment (jsumners) [inline index.js]: Wouldn't it be better to `Object.assign({}, defaultCtx, data)`? This way you do not mutate the object passed in.

Comment (farzer) [reply index.js]: fixed

**Comment by farzer** [comment]
added the readme

File: README.md [hunk 2]
```diff
@@ -76,6 +76,24 @@ fastify.register(require('point-of-view'), {
 })
 ```
 
+if you want to set a default context that the variable can be using in each view:
```
Comment (mcollina) [inline README.md]:
```suggestion
If you want to set a default context that the variable can be using in each view:
```

**Pull request merged**